### PR TITLE
Adding README for grafana component

### LINKED ISSUE #31: Add README for Grafana

### PR DESCRIPTION
**Pull request by crobby**
Closes #31

### CODE REVIEW

File: grafana/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+This installation creates a grafanadashboard instance that is meant to view data from the strimzi-kafka installation
+
+## Grafana Datasource
+
+This instance creates a grafanadatasource instance that scrapes data from our Prometheus
+installation and is required for the grafanadashboard instance mentioned above.
+
+# Installation
+To install Grafana add the following to the `kfctl` yaml file.
```
Comment (vpavlin) [inline grafana/README.md:25]: I am not sure if `kfctl` is the right name - the resource in the file is `KFDef`, so maybe use that?

File: grafana/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+```
+  - kustomizeConfig:
+      parameters:
+      - name: namespace
```
Comment (vpavlin) [inline grafana/README.md:30]: Do not recommend using the `namespace` - that basically hardcodes the namespace for the component

**Comment by crobby** [comment]
rebased to hopefully get the tests to pass

**Comment by vpavlin** [comment]
still /lgtm :)

**Pull request merged**